docs/content/patterns/alz/deploy/Customize-Policy-Assignment.md

```diff
@@ -35,11 +35,11 @@ The parameter file contains the same default values as listed in our documentati
         "policyAssignmentParametersCommon": {
             "value": {
                 "ALZMonitorResourceGroupName": {
-                    "value": "rg-alz-monitor"
+                    "value": "rg-amba-monitoring-001"
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "alz-monitor"
+                        "_deployed_by_amba": true
```

Comment:
We should keep "Project": "amba-monitoring" as the example value. This is for customers to change should they wish to add tags to the RG.